package.json

```diff
@@ -11,7 +11,7 @@
     "remove-locks": "lerna exec 'rm package-lock.json'",
     "install-locks": "lerna exec 'npm i --package-lock-only --no-audit' --parallel --no-sort",
     "lint": "eslint example packages/api packages/checksum packages/cmrjs packages/cmr-client packages/common packages/deployment packages/ingest tf-modules/s3-replicator tasks",
-    "test": "nyc lerna run --ignore cumulus-integration-tests test",
+    "test": "nyc lerna run --concurrency 1 --ignore cumulus-integration-tests test",
```

Comment:
I'm not convinced CI is running on 1 core, the times *were* altered when this was set to 1.  

Will not hold up approval, but I'm expecting this one to come back eventually.

Reply:
I'm not convinced it's 1 by default in CI either, but I do suspect that whatever resources we're being given for free in CI is probably less than what we have available on our laptops.